synfig-studio/src/gui/preview.cpp

```diff
@@ -456,7 +456,7 @@ Widget_Preview::Widget_Preview():
 	toolbar->pack_start(*next_framebutton, Gtk::PACK_SHRINK, 0);
 
 	//spacing
-	Gtk::Alignment *space = Gtk::manage(new Gtk::Alignment());
+	Gtk::Fixed *space = Gtk::manage(new Gtk::Fixed);
```

Comment:
No need to use `Fixed` here. You can just remove it.

This is `Preview` window toolbar, you can open this window using `File->Preview-> (click Preview button)`

Reply:
@ice0 Thank You ! but how will I come to know it myself that which file is meant for what ? i.e. how did you came to know that preview.cpp is for preview window toolbar in ``` file->preview->previewButton ``` ?  it would  help a lot if I can see these changes that I am making ! that's why I am having so much doubts.

and thank you for telling me about gtk inspector ! Its great ! Trying to get hang of that tool :)

Thank You So Much  [^_^]

Reply:
I searched for usages of the `Preview` class and found that it is being used from `Dialog_Preview::set_preview(etl::handle<Preview> prev)`. After that you need to search for usages of `Dialog_Preview` etc. But at that point I did not do any further search because I already knew where the preview dialog was (this is just an experience).

Reply:
Understood ! Thank You so much for help ^_^